### ISSUE TITLE
REST interface

### ISSUE THREAD

**Issue by goodmami** [delph-in/pydelphin#66]
Possibly include a REST interface (e.g. `delphin.interfaces.rest`) for parser output (including derivations, MRS, etc.).

See here: http://moin.delph-in.net/ErgApi

Also see this discussion: http://lists.delph-in.net/archives/developers/2016/002271.html

If a server (e.g. [Bottle](http://bottlepy.org)) is required, then maybe this should be a separate module from pyDelphin. If it does belong, however, then the results from `delphin.interfaces.ace` should probably use this API (at least the Python side of it).

The EDS output would depend on #25.

-----
**Edit:** pyDelphin can provide a REST client, which formats the request, sends it to a server, receives the response, and interprets the values of the response, but it's probably out-of-scope to provide a REST server as well. The client thus requires codecs for some other formats (e.g. the JSON formats; see #68), but it's not expected to interpret all (e.g. the LaTeX formats).
